kubemarine/resources/configurations/defaults.yaml

```diff
@@ -19,7 +19,7 @@ services:
   kubeadm:
     apiVersion: kubeadm.k8s.io/v1beta2
     kind: ClusterConfiguration
-    kubernetesVersion: v1.24.0
+    kubernetesVersion: v1.24.2
```

Comment:
Though highly unlikely, but there theoretically can be the one who omitted the `kubernetesVersion` parameter. After that one takes new version of Kubemarine, it becomes to think that the cluster is based on v1.24.2, not v1.24.0. To resolve this, we can develop the patch that will migrate the inventory, or make the `kubernetesVersion` parameter mandatory.

Reply:
mmm, let's leave this "highly unlikely" case to be fixed manually